### PR TITLE
Structural baseline updates (cleaned up)

### PR DESCRIPTION
**Pull request by ahuynhMITRE**
## 🗣 Description ##

Cleaned up Pull request for updating the baseline markdown files with some of the identified  "structural" changes to serve as a starting point for the team to start on "substantive" changes and in support of the automation of the markdown files to json (issue #209). Following changes include:

- REMOVE All Numbering from “Introductions”, “Assumptions”, and “Baseline” sections of the markdown. 
- ADD New baseline versioning schema to each policy.
- ADD “Last Modified” section below “Rationale” Section.
- UPDATE Group Name to remove "SHALL" and "SHOULD" language.
- UPDATE all relative references (e.g., "see above policies”) with new policy IDs.
- updated all policy IDs for exchange from MS.EXCHANGE to MS.EXO
- Updated the markdown file name exchange.md to exo.md

## 💭 Motivation and context ##

This change is required to support testing of Issue #209 and is in support of creating a "starting point" for the issues associated with the "substantive" changes for each M365 product. Many of the updates also support partially resolving the "Markdown document alignment issues"  opened for each product. 

## 🧪 Testing ##

Changes can be tested by confirming script used in Issue #209 is supported. Markdown files are not corrupt and open as expected.

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - *eschew scope creep!*
- [x] All relevant repo and/or project documentation has been updated to reflect the changes in this PR.

## ✅ Pre-merge checklist ##

- [x] Revert dependencies to default branches.
- [x] Finalize version.

## ✅ Post-merge checklist ##

- [ ] Validate post-merge checks passed on merged branch

### CODE REVIEW

File: baselines/aad.md [hunk 1]
```diff
@@ -784,32 +822,36 @@ identity compromise.[^1]
...
-### 2.13.1 Policy
-
-- MFA SHALL be required for user access to highly privileged roles.
+### Policies
+#### MS.AAD.13.1v1
+MFA SHALL be required for user access to highly privileged roles.
+- _Rationale:_ TODO
+- _Last modified:_ June 2023
 
 <!-- -->
```
[the remark this answers is not in the document]
Comment (ahuynhMITRE) [reply baselines/aad.md:857]: updated!

File: baselines/powerbi.md [hunk 2]
```diff
@@ -411,27 +378,7 @@ permissions.
```
[the remark this answers is not in the document]
Comment (ahuynhMITRE) [reply baselines/powerbi.md:378]: updated!

**Review by ahuynhMITRE — COMMENTED** [review]
great finds Addam, have made the appropriate updates!

**Review by tkol2022 — APPROVED** [review]
This was a lot of work. Nice job!

**Pull request merged**